src/core/app/init.ts

```diff
@@ -17,56 +19,59 @@ export async function appInit() {
 
   const initialState = urlEncoder.decode<UrlData>(document.location.search.slice(1));
 
-  // Stage - 1
-  // get application ID from URL or API
-  // load custom app config from api
-  const appId = initialState.app || (await apiClient.get<string>('/apps/default_id'));
-  if (!appId) {
-    // cannot continue without appId
-    throw new Error('Error getting application ID');
-  }
-  initialState.app = appId;
+  const localAuthToken = authClientInstance.checkLocalAuthToken();
 
-  const appInfo = await apiClient.get<AppInfoResponse>(
-    `/apps/${appId}`,
-    undefined,
-    false,
+  const appInfo = await apiClient.get<AppConfiguration>(
```

Comment:
naming suggestion:
we already have `appConfig` on this file, `appInfo` sounds very similar to it
how about smth like `remoteAppInfo` or ``remoteAppData`?